libevmcore/Instruction.cpp

```diff
@@ -82,6 +82,8 @@ const std::map<std::string, Instruction> dev::eth::c_instructions =
 	{ "SSTORE", Instruction::SSTORE },
 	{ "JUMP", Instruction::JUMP },
 	{ "JUMPI", Instruction::JUMPI },
+	{ "JUMPV", Instruction::JUMP },
```

Comment:
I think it would be better to add these to a separate table - they are not official opcodes.